README.md

```diff
@@ -124,13 +124,13 @@ npm test -- --ver=alpha --invert --grep="System"
 ## Test with core
```

Comment:
should "specifying the core path" section be removed?

Reply:
👍